miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -346,6 +350,10 @@ export.account_vault_add_asset
     exec.authenticate_account_origin
     # => [ASSET]
 
+    # TODO: we execute `push.1 drop` before `emit` as decorators are not supported without other
+    # instructions - see: https://github.com/0xPolygonMiden/miden-vm/issues/1122
+    push.1 drop emit.ACCOUNT_VAULT_BEFORE_ADD_ASSET_EVENT 
```

Comment:
Let's use `push.0 drop` here and in other places (unless there is a good reason to use `push.1`?).